aviary/interface/methods_for_level2.py

```diff
@@ -1711,12 +1695,41 @@ def add_design_variables(self):
             for dv_name, dv_dict in dv_dict.items():
                 self.model.add_design_var(dv_name, **dv_dict)
 
-        if self.mission_method in (HEIGHT_ENERGY, SOLVED_2DOF):
+        if self.mission_method is SOLVED_2DOF:
             optimize_mass = self.pre_mission_info.get('optimize_mass')
             if optimize_mass:
                 self.model.add_design_var(Mission.Design.GROSS_MASS, units='lbm',
                                           lower=100.e2, upper=200.e3, ref=135.e3)
 
+        elif self.mission_method is HEIGHT_ENERGY:
+            if self.problem_type is ProblemType.SIZING:
+                self.model.add_design_var(Mission.Design.GROSS_MASS, units='lbm',
+                                          lower=100e2, upper=200e3, ref=135e3)
+
+                self.model.add_design_var(Mission.Summary.GROSS_MASS, units='lbm',
```

Comment:
What is the difference between Design Gross Mass, and Summary Gross Mass? Are they truly independent variables that the optimizer should control separately?

Reply:
Mission.Design.GROSS_MASS is the gross weight for the design mission, which drives the structural requirements->OEW->GTOW as part of the design loop.
Mission.Summary.GROSS_MASS is the gross mass at the start of the current mission. For the design mission, this is equal to the design gross mass, for any other mission it may or may not be. For non-design missions, the design gross mass is held constant, but the summary gross mass is allowed to change. This prevents the OEW from resizing

Reply:
also, I believe Kaushik is planning to combine this with the 2DOF section below, which has some comments explaining the different constraints/DVs

Reply:
Thanks. I think this makes sense now.